system-setup.py

```diff
@@ -40,12 +40,12 @@ def fedora(self):
         self.run("%s/bin/getgcc" % READIES)
 
     def macos(self):
+        self.run("pip3 install semantic_version")
         self.install_gnu_utils()
         self.install("pkg-config")
 
         # for now depending on redis from brew, it's version6 with TLS.
-        #self.run("{PYTHON} {READIES}/bin/getredis -v 6 --force".format(PYTHON=self.python, READIES=READIES))
-        self.install("redis")
+        self.run("{PYTHON} {READIES}/bin/getredis -v 6 --force".format(PYTHON=self.python, READIES=READIES))
```

Comment:
We just switched from `getredis` to `brew install redis` a few commits ago:
https://github.com/RediSearch/RediSearch/commit/7a389122ed507f801da4fd1510567afa79b9997d

I don't mind giving it a try, just notice `getredis -v 6` now gets Redis v6.2.x, so we might be failing on Redis 6.0.x for some mysterious reason.

Reply:
The reason we fail is the header in getredis. Specifically it checks for python3 support and then falls back to python2. The issue is that the mac build for RS has python3, but not the module.

We move to brew, because (environmentally) it brought the latest redis - but in CI it brings redis-5 not 6. If we fail on redis6 due to support, I view that as an improvement :)

Reply:
Ok, let's do it.